stacks/java-websphereliberty/devfile.yaml

```diff
@@ -18,9 +18,8 @@
 schemaVersion: 2.1.0
 metadata:
   name: java-websphereliberty
-  version: 0.9.0
-  displayName: 'WebSphere Liberty Maven'
-  description: Java application Maven-built stack using the WebSphere Liberty runtime
+  displayName: WebSphere Liberty Maven
+  description: Java application based OpenJ 9 Java 11 and Maven 4.0, using the WebSphere Liberty runtime 22.0.0.1
```

Comment:
Similar to the java-openliberty-gradle. Mentioning both OpenJDK 9 and Java 11 is in conflict.